cypress/support/steps/main.steps.ts

```diff
@@ -95,15 +106,22 @@ export const borrow = (
 
   return describe(`Borrow process for ${_shortName}`, () => {
     skipSetup({ skip, updateSkipStatus });
-    it(`Open ${_shortName} borrow popup view`, () => {
-      doSwitchToDashboardBorrowView();
-      cy.wait(4000);
-      cy.get(`[data-cy='dashboardBorrowListItem_${_shortName.toUpperCase()}']`)
-        .contains('Borrow')
-        .should('not.be.disabled')
-        .click();
-      cy.get(`[data-cy=Modal] h2:contains("Borrow ${_shortName}")`).should('be.visible');
-    });
+    if (!isDetailsPage) {
+      it(`Open ${_shortName} borrow popup view`, () => {
+        doSwitchToDashboardBorrowView();
+        cy.wait(4000);
+        cy.get(`[data-cy='dashboardBorrowListItem_${_shortName.toUpperCase()}']`)
+          .contains('Borrow')
+          .should('not.be.disabled')
+          .click();
+        cy.get(`[data-cy=Modal] h2:contains("Borrow ${_shortName}")`).should('be.visible');
+      });
+    } else {
+      it('Open via details page', () => {
+        cy.contains('Your info').click();
+        cy.contains('Borrow').click();
+      });
+    }
```

Comment:
i guess that u don't need to change this one, cause u never use this functionality